controllers/Index.php

```diff
@@ -175,8 +175,15 @@ public function onDelete()
         $this->validateRequestTheme();
 
         $type = Request::input('objectType');
+        $object = $this->loadObject($type, trim(Request::input('objectPath')));
 
-        $deletedObjects = $this->loadObject($type, trim(Request::input('objectPath')))->delete();
+        $deletedObjects = $object->delete();
+
+        /*
+         * Extensibility
+         */
+        Event::fire('pages.object.delete', [$this, $object, $type]);
```

Comment:
@2mt-heuser Would you mind adding a comment explaining the event and its arguments? Similar to this:
https://github.com/octobercms/october/blob/315e2dcac57741df66efa07a001f08730e7bc943/modules/cms/classes/CmsObject.php#L140

Reply:
Added comments to the two new delete event fires as well as the save one that only had "extensibility" in there.